### PR TITLE
Draft On test 2024 08 05

### PR DESCRIPTION
**Pull request by nOester**
Moving Links (Contact & disclaimer to ngm-side-bar)

### CODE REVIEW

**Comment by oesterli** [comment]
Hi @vladyslav-tk
Could you have a quick look at this PR and the error. If I understand the errors correctly its only linter errors.
I pushed from vscode and didn't change the most of the lines mentioned in the erros. How can I fix it?

**Comment by vladyslav-tk** [comment]
Screenshot 2024-08-15 at 17 19 01


Should we add a margin here?

**Comment by vladyslav-tk** [comment]
@nOester I put some comments, please have a look and let me know if you have questions or if I should handle it

File: ui/src/elements/ngm-side-bar.ts [hunk 1]
```diff
@@ -266,15 +266,23 @@ export class SideBar extends LitElementI18n {
...
         <div class="toolbar-settings">
-          <label>${i18next.t('lsb_debug_tools')}</label>
-          <div class="ngm-checkbox ngm-debug-tools-toggle ${classMap({active: this.debugToolsActive})}"
-               @click=${() => (<HTMLInputElement> this.querySelector('.ngm-debug-tools-toggle > input')).click()}>
-            <input type="checkbox" ?checked=${this.debugToolsActive} @change="${this.toggleDebugTools}">
-            <span class="ngm-checkbox-icon">
-              </span>
-            <label>${i18next.t('lsb_cesium_toolbar_label')}</label>
+            
+            <!-- New container start -->
```
[the remark this answers is not in the document]
Comment (nOester) [reply ui/src/elements/ngm-side-bar.ts:270]: @vladyslav-tk Yes, you are right. I'm going to remove the comments and push the branch again.

File: ui/src/ngm-app.ts [hunk 2]
```diff
@@ -514,10 +514,10 @@ export class NgmApp extends LitElementI18n {
                          @toggleProjectSelector=${() => this.showProjectSelector = !this.showProjectSelector}>
               </view-menu>
             </div>
-            <a class="contact-mailto-link" target="_blank"
+            <!-- <a class="contact-mailto-link" target="_blank"
```
[the remark this answers is not in the document]
Comment (nOester) [reply ui/src/ngm-app.ts:517]: done

File: ui/src/style/ngm-side-bar.css [hunk 3]
```diff
@@ -76,6 +76,15 @@ ngm-side-bar {
   overflow-y: auto;
 }
 
+/* New css class start */
```
[the remark this answers is not in the document]
Comment (nOester) [reply ui/src/style/ngm-side-bar.css:79]: done

File: ui/src/style/index.css [hunk 4]
```diff
@@ -513,9 +513,9 @@ ngm-voxel-filter .ngm-action-btn.ui.button {
   position: absolute;
   z-index: 2;
   left: 5px;
-  font-size: 12px;
-  color: #fff;
-  text-shadow: 0 0 2px #000;
+  /* font-size: 12px; */
```
[the remark this answers is not in the document]
Comment (nOester) [reply ui/src/style/index.css:516]: done

**Comment by nOester** [comment]
Hi @vladyslav-tk

File: ui/src/elements/ngm-auth.ts [hunk 5]
```diff
@@ -63,7 +63,7 @@ export class NgmAuth extends LitElementI18n {
       <div class="ngm-user ${classMap({'ngm-active-section': !!this.user})}"
            @click=${!this.user ? this.login : this.logout}>
         <div class="ngm-user-icon"></div>
-        ${!this.user ? i18next.t('lsb_login') : i18next.t('lsb_logout')}
+        <!-- ${!this.user ? i18next.t('lsb_login') : i18next.t('lsb_logout')} -->
```
Comment (vladyslav-tk) [inline ui/src/elements/ngm-auth.ts:66]: The "Login" text is hidden now. Is it expected?

Comment (nOester) [reply ui/src/elements/ngm-auth.ts:66]: @vladyslav-tk 
Yes, that's correct. As for other applications, the log button does not show a text label.

**Review by vladyslav-tk — APPROVED** [review]

File: ui/src/elements/ngm-auth.ts (hunk 5 above)
Comment (nOester) [reply ui/src/elements/ngm-auth.ts]: Done

**Pull request merged**